README.md

```diff
@@ -221,7 +221,8 @@ Some platforms have specific requirements:
 For building, it requires:
 
 * Gradle 5.6
-* Kotlin 1.5+ (1.7+ for native)
+* Kotlin 1.8+ for JVM projects
+* Kotlin 1.9+ for native projects
```

Comment:
Are we sure native doesn't work with 1.8 these days?

Reply:
Looks like not, I can't build a project with `1.8.22` with:

```
w: skipping /Users/bod/.gradle/caches/modules-2/files-2.1/com.apollographql.apollo3/apollo-normalized-cache-sqlite-macosarm64/3.8.3/689b3b4527baf866a88cb0559ef1772c7b8216c7/apollo-normalized-cache-sqlite.klib. Incompatible abi version. The current default is '1.7.0', found '1.8.0'. The library produced by 1.9.20 compiler
e: Could not find "/Users/bod/.gradle/caches/modules-2/files-2.1/com.apollographql.apollo3/apollo-normalized-cache-sqlite-macosarm64/3.8.3/689b3b4527baf866a88cb0559ef1772c7b8216c7/apollo-normalized-cache-sqlite.klib" in [/Users/bod/gitrepo/apollo-kotlin-template-mpp, /Users/bod/.konan/klib, /Users/bod/.konan/kotlin-native-prebuilt-macos-aarch64-1.8.22/klib/common, /Users/bod/.konan/kotlin-native-prebuilt-macos-aarch64-1.8.22/klib/platform/macos_arm64]
```

Although I'm a bit confused by the "1.7.0" mention in the message.

Reply:
I opened https://youtrack.jetbrains.com/issue/KT-66755/Native-add-support-for-languageVersion to try to understand this better. 

LGTM in all cases. Let's ask for 1.9 since we know this will work. K/N consumers are probably already on 1.9 anyways.